src/utilities/isPrimitiveValueExpression.ts

```diff
@@ -1,5 +1,6 @@
 // @flow
 
-export default (maybe: *): boolean %checks => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export default (maybe: any): boolean => {
```

Comment:
We could make this a type guard to provide equivalent functionality
```suggestion
export default (maybe: any): maybe is string | number | boolean | null => {
```